src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
```

Comment:
The code provided seems fine and does not raise any issues as such. Here is a summary of the key points in the code: